.travis.yml

```diff
@@ -85,8 +85,14 @@ before_install:
     - tools/check_sdist.py
 
 install:
-    - python setup.py develop
     - ccache --show-stats
+    # Test installing without cython using the sdist
+    - if [[ "$INSTALL_FROM_SDIST" == "1" ]]; then
+        pip uninstall cython -y ;
```

Comment:
minor nitpick: semi-colons are unnecessary here

Reply:
ok.

Reply:
I think they are necessary. Travis unwraps all of these into a "single line command". Really annoying, but c'est la vie.